nearlib/test/integration.test.js

```diff
@@ -159,13 +156,11 @@ const waitForContractToDeploy = async (deployResult) => {
     );
 };
 
-const waitForNonceToIncrease = async (initialAccount) => {
-    await callUntilConditionIsMet(
-        async () => { return await account.viewAccount(initialAccount['account_id']); },
-        (response) => { return response['nonce'] != initialAccount['nonce'] },
-        "Call view account until nonce increases"
-    );
-};
+function sleep(time) {
```

Comment:
alternatively this also works in Node.js:
```js
const sleep = require('utils').promisify(setTimeout);
```
this approach should generally work for various functions taking callbacks